app/scripts/modules/kubernetes/src/v2/pipelines/stages/patchManifest/patchManifestConfig.html

```diff
@@ -0,0 +1,50 @@
+<div ng-if="ctrl.state.loaded" class="clearfix" class="clearfix">
+  <div class="alert alert-warning">
+    <strong>Warning!</strong> This stage is under active development and is subject to change.
+  </div>
+
+  <div class="container-fluid form-horizontal">
+    <h4>Resource to Patch</h4>
+    <div class="horizontal-rule"></div>
+    <kubernetes-manifest-selector selector="ctrl.$scope.stage"></kubernetes-manifest-selector>
+
+    <h4>Patch Content</h4>
+    <ng-form name="kubernetesManifestSource">
+      <stage-config-field label="Manifest Source" help-key="kubernetes.manifest.source">
+        <label class="sm-label-right">
+          <input type="radio" ng-model="ctrl.$scope.stage.source" value="{{ctrl.textSource}}">
+          {{ ctrl.textSource | robotToHuman }}
+        </label><br/>
```

Comment:
Rather than using a `br` to stack these (if that's what's happening), wrap each of these in a `<div class="radio">...</div>`